test_cases/test_dojo.py

```diff
@@ -239,6 +243,37 @@ def test_load_people_invalid_entry(self):
 		result = self.the_dojo.load_people("LoadFile")
 		self.assertEqual(result, "Invalid entry!")	
 
+	def test_save_state_database_file_exists(self):
+		self.the_dojo.save_state("sampleDB")
+		self.assertTrue(os.path.isfile("sampleDB.db"))
+		os.remove("sampleDB.db")
+
+	def test_save_state_successfuly_saves_data_to_DB(self):
+		self.the_dojo.create_room("office", "Red")
+		self.the_dojo.save_state("sampleDB")
+		engine = create_engine('sqlite:///sampleDB.db')
+		Base.metadata.bind = engine
+		DBSession = sessionmaker(bind=engine)
+		session = DBSession()
+		#check if data saved exists in db
+		existing = session.query(RoomModel).all()
```

Comment:
You will need to check if the data has been stored correctly. What if your function has a bug that stores only one room? What if living spaces are not being picked by your function?